itou/www/siae_evaluations_views/views.py

```diff
@@ -142,6 +143,7 @@ def institution_evaluated_job_application(
     )
     context = {
         "evaluated_job_application": evaluated_job_application,
+        "evaluated_siae_state": evaluated_job_application.evaluated_siae.state,
```

Comment:
to be fixed

Reply:
laisse juste un commentaire ici (et potentiellement dans le test a l'endroit du assert num queries) pour dire que l'on ne sait pas vraiment pourquoi les requetes sont faites, que l'on a tenté de passer l'objet mais que en gros la cached_property ne fonctionne pas.

j'ai d'autres idées mais on va cloturer la PR, ça suffit comme ça